tests/test_datasets/test_dataset.py

```diff
@@ -220,6 +222,19 @@ def test_vaihingen():
     assert len(test_dataset) == 1
 
 
+def test_synapse():
+    test_dataset = SynapseDataset(
+        pipeline=[],
+        data_prefix=dict(
+            img_path=osp.join(
+                osp.dirname(__file__),
+                '../data/pseudo_synapse_dataset/img_dir'),
+            seg_map_path=osp.join(
+                osp.dirname(__file__),
+                '../data/pseudo_synapse_dataset/ann_dir')))
+    assert len(test_dataset) == 5
```

Comment:
Not necessary to upload too many files in pesudo dataset folder. I think 2 or 3 is enough.